website/docs/addon_maya_arnold_artist.md

```diff
@@ -21,20 +21,20 @@ When creating the instance, two objectsets are created; `content` and `proxy`. M
 In order to utilize operators and proxies, the content and proxy nodes need to share the same names (including the shape names
 and CBIDs). This is done by parenting the content and proxy nodes into separate groups. For example:
 
-![Arnold Scene Source](assets/maya-arnold_scene_source.png)
+![Arnold Scene Source](assets/maya/arnold_artist/arnold_scene_source.png)
```

Comment:
I think splitting the images into these workflow folder might be too much. Having `admin` and `artist` would be enough and it pretty clear where images should go.